pkg/controller/perconaservermongodb/psmdb_controller.go

```diff
@@ -1211,3 +1308,26 @@ func OwnerRef(ro runtime.Object, scheme *runtime.Scheme) (metav1.OwnerReference,
 		Controller: &trueVar,
 	}, nil
 }
+
+func (r *ReconcilePerconaServerMongoDB) hasMongodCustomConfiguration(cr *api.PerconaServerMongoDB, rs *api.ReplsetSpec) (bool, error) {
+	if rs.Configuration != "" {
+		return true, nil
+	}
+
+	var name = psmdb.MongodConfigCMName(cr.Name, rs.Name)
```

Comment:
let's use this function result directly at `Name` field

Reply:
this function is going to check Secrets as well, so the name will be used twice in the next PR